### ISSUE TITLE
ptpAnnounceReceiptTimeout description in gendoc

### ISSUE THREAD

**Issue by eduardoyusta** [openBackhaul/synchronization#38]
Taking as a reference the latest shared gendoc file (v.220404.1830_AlWe_TH), the current description indicates the parameter represents the exponent of a 2^N number like some others in the PTP_Pac.

In the case of the announceReceiptTimeout, it represents directly an integral number of announceIntervals to be used used for timeout purposes in the port protocol operation (e.g. 1588-2019 7.7.3.1 and 8.2.15.4.2). This way the current comment does not apply and needs modification.

Datatype is specified as Uint8 (0-255). IEEE default profile sets 3 as inicialization value and 2-10 as range for implementation (1588-2019 I.3.2). ITU profiles specify 2, 3 or refers generally to PTP specification for ranges depending on profile and element role (8275.1 table A.5, 8275.2 table A.5, 8265.1 table A.5).

**Comment by DanielaSpreafico** [comment]
I agree: the announceReceiptTimeout is not an exponenet but the timeout to use for the annnonce messages . It must be in the range 2..10.

**Comment by openBackhaul** [comment]
**Proposal to the 5G-xhaul call on 7th of September 2022:**
Datatype shall be changed from INT16 to INT8.
Unit (currently ms) shall be removed.
Comment shall change from
"It is used for fault detection of PTP announce messages. Uses log base 2 values. This value belongs to the exponent 2 - 10."
to
"It is used for fault detection of PTP announce messages. The expected value should be in the range 2 to 10.".

**Comment by Nzein** [comment]
In G.8275.1, the range is 3 - z, where z is For Further Study.
Therefore, the IEEE default profile is 2 - 10, but we believe it is safer not to fix it to 2 - 10 when using the telecom profile.

**Comment by demx8as6** [comment]
Very unlikely that z will be > 10.
The range 2 -10 is currently implemented. Avoiding operational issue we better not restrict in 5G-xHaul model than 2-10.

**Comment by demx8as6** [comment]
**Decision made at the 5G-xhaul call on 7th of September 2022:**
Datatype shall be changed from INT16 to INT8.
Unit (currently ms) shall be removed.
Comment shall change from
"It is used for fault detection of PTP announce messages. Uses log base 2 values. This value belongs to the exponent 2 - 10."
to
"It is used for fault detection of PTP announce messages. The expected value should be in the range 2 to 10.".
